mixins/localize-mixin.js

```diff
@@ -49,7 +49,12 @@ export const LocalizeMixin = superclass => class extends superclass {
 	}
 
 	localize(key) {
-		return this._computeLocalize(this.__language, this.__resources, key);
+		const args = {};
+		for (let i = 1; i < arguments.length; i += 2) {
+			args[arguments[i]] = arguments[i + 1];
+		}
```

Comment:
I'm guessing this was caching the first value previously? Unfortunately we haven't [pulled in the old tests](https://github.com/BrightspaceUI/localize-behavior/blob/master/test/d2l-localize-behavior.html) yet... although not sure they would have caught this.

Reply:
Exactly, and the function where this was being run didn't even have the extra parameters. I caught this as soon as I tried to translate anything with a parameter.